### DISCUSSION TITLE
[ReverseProxyPlugin] Redirect HTTP request to an HTTPS server with one field added to the request headers

### DISCUSSION THREAD

**Discussion by deeplook** [abhinavsingh/proxy.py#520] [Q&A] [answered]
Looking through the list of existing platform, I get a sense of there should be something I want already, but then it's still hard to locate it. Maybe somebody can help? I'm looking for a plugin that redirects HTTP requests to an HTTPS server with one field added to the request headers. I can make requests to the HTTPS server with the `requests` package already without passing any certificates, but I need a "frontend" that accepts calls without a header.

Let me add that I've looked at the tests for the existing plugins in the testsuite, but there is so much mocking going on that it's a bit hard to detect the essentials.

**Comment by abhinavsingh** [comment, marked as answer]
To conclude this discussion, `ReverseProxyPlugin` can do this out-of-box.
